tests/test_openjdk.py

```diff
@@ -67,23 +67,42 @@ def test_jdk_version(container, java_version):
     indirect=["container_per_test"],
 )
 @pytest.mark.parametrize(
-    "test_to_run, expected_strings",
+    "test_to_run, expected_strings, expected_exit_status, java_params",
     [
         (
             "threads_concurrency_and_sleep",
             ["I am the thread 1", "I am the thread 2"],
+            [0],
+            "",
         ),
-        ("time", ["All OK"]),
+        ("time", ["All OK"], [0], ""),
         (
             "files_and_directories",
             ["All OK"],
+            [0],
+            "",
         ),
+        (
+            "memory",
+            ["Iteration: (2)", "OutOfMemoryError"],
+            [1],
+            "-Xmx10M",
+        ),
+        (
+            "memory",
+            ["Iteration: (45)", "OutOfMemoryError"],
+            [1],
+            "-Xmx100M",
```

Comment:
Funny Java, that we need twice the memory as we actually allocate? Or is it copied when we call
```
v.add(b);
```
??

Reply:
The idea of this test is to check if parameter Xmx works effectively, so first I allocate 10Mb and then 100Mb seeing witch iteration we have. Maybe I two tests for the same is not necessary

Reply:
I removed the second one

Reply:
ok, this wasn't my intention, sorry for the confusion.